.gitignore

```diff
@@ -101,3 +101,5 @@ version.ml
 version.texi
 .cargo
 !src/vendor/**
+**/target/rls/debug/
+/src/target/
```

Comment:
@rkruppe 

`/src/target/` appears when manually running `cargo build` inside the crates like `/src/libsyntax/`. 

`**/target/rls/debug/` appears when opening any folder in VSCode with kalitaalexey's `vscode-rust` extension, configured to use RLS.